src/utils/presence.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-mixed-spaces-and-tabs */
```

Comment:
Why do you need this? (I know eslint will warn you about it, but can't you fix it?)

Reply:
Just tested it. And I have no idea why this is happening lol.